### ISSUE TITLE
Support for tags

### ISSUE THREAD

**Issue by mad4633** [MikeSouza/serverless-deployment-bucket#12]
Would it possible to support for tags?

**Comment by MikeSouza** [comment]
Should be easy enough to do. Sure.

**Comment by jdrydn** [comment]
@MikeSouza If this isn't something you're currently working on, do you mind if I create a PR with the relevant changes for you to review?

**Comment by MikeSouza** [comment]
@jdrydn Yes please feel free to make a PR. I am quite busy right now and would welcome the help. Please make sure to write unit tests and maintain the 100% coverage.
